index.ts

```diff
@@ -4,17 +4,24 @@ import dns from "node:dns";
 import { ActivityType, GatewayIntentBits } from "discord.js";
 import "dotenv/config";
 import pkg from "./package.json" assert { type: "json" };
-import { GlobalFonts } from "@napi-rs/canvas";
 import { login, client } from "strife.js";
-import { Chart } from "chart.js";
 import constants from "./common/constants.js";
 
+let GlobalFonts = null,
```

Comment:
Why do these need to be defined outside of the if?

Reply:
That's a great point, they don't anymore.